clients/admin-ui/cypress/e2e/consent-settings.cy.ts

```diff
@@ -183,6 +184,7 @@ describe("Consent settings", () => {
         cy.intercept("/api/v1/config?api_set=false", { body: DEFAULT_CONFIG });
         cy.intercept("/api/v1/config?api_set=true", { body: {} });
         cy.visit(GLOBAL_CONSENT_CONFIG_ROUTE);
+        cy.getByTestId("section-GPP U.S.").should("exist");
```

Comment:
noticed that Cypress wasn't waiting for the page to even load before checking for "not.exist" so it was always passing no matter what. This change ensures DOM is loaded before checking.